### PR TITLE
Add template for pull request

### PR DESCRIPTION
**Pull request by naokidney**
# Description

We can create a template for pull request, and I made it 😄 

https://docs.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository

### CODE REVIEW

**Comment by naokidney** [comment]
Can you check the template I made, and please approve it if it's okay

**Comment by naokidney** [comment]
Here's how to approve.
https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/approving-a-pull-request-with-required-reviews

 Of course, if you wanna ask something, or there's problem about my code, please don't `Approve` it. Please `Comment` or `Request Change` instead.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Description
+
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project(Expample)
```
Comment (kristinejosami) [inline .github/PULL_REQUEST_TEMPLATE.md]: please correct the spelling of the word "(Expample)" to (Example)"

**Review by kristinejosami — CHANGES_REQUESTED** [review]
kindly convert the template's name to small letters(snakecase)

**Comment by kristinejosami** [comment]
> # Description
> We can create a template for pull request, and I made it 😄
> 
> https://docs.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository

I wanna use emoji when replying. Is it built in to your keyboard? hahahahaha I know it isn't but where can I see emoji icon on github?

**Comment by kristinejosami** [comment]
> > # Description
> > We can create a template for pull request, and I made it 😄
> > https://docs.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository
> 
> I wanna use emoji when replying. Is it built in to your keyboard? hahahahaha I know it isn't but where can I see emoji icon on github?

😄 ❤️ 😆 🤣 HAHAHAHAHAHAHAHAHA I searched it. I wanna insert emojis to convey my feelings when writing a message lol. 😆 🤣 😚 😍 😝 💩 
https://gist.github.com/rxaviers/7360908

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (naokidney) [reply .github/PULL_REQUEST_TEMPLATE.md]: Good joooob 😭

Comment (naokidney) [reply .github/PULL_REQUEST_TEMPLATE.md]: Usually, I'm the one who fix this. But this time, can you please give me a suggested change?
https://haacked.com/archive/2019/06/03/suggested-changes/

Comment (naokidney) [reply .github/PULL_REQUEST_TEMPLATE.md]: @kristinejosami

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Description
+
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project(Expample)
```
Comment (kristinejosami) [inline .github/pull_request_template.md]:
```suggestion
- print("My code follows the style guidelines of this project(Example)")
```

Comment (naokidney) [reply .github/pull_request_template.md]: @kristinejosami  Oh, sorry, I'm using markdown...
Can you check it and give me another suggestion?

https://guides.github.com/features/mastering-markdown/

Comment (kristinejosami) [inline .github/pull_request_template.md]:
```suggestion
- [ ] My code follows the style guidelines of this project(Example) 
```

**Comment by naokidney** [comment]
@kristinejosami please review again 😸

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Description
+
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project(Example)
```
Comment (kristinejosami) [inline .github/pull_request_template.md]:
```suggestion
- [ ] My code follows the style guidelines of this project example
```

Comment (kristinejosami) [reply .github/pull_request_template.md]: @naokitcut please check 😸

**Comment by kristinejosami** [comment]
LGTM 👍

**Review by kristinejosami — COMMENTED** [review]
LGTM 👍

**Comment by naokidney** [comment]
@kristinejosami can you approved it, not comment 😄

**Review by kristinejosami — APPROVED** [review]

**Comment by naokidney** [comment]
@kristinejosami Thankies 😄

**Pull request merged**